nxt-messaging/src/messaging/ByteUtility.java

```diff
@@ -23,6 +23,8 @@ public static int bytesToInt(byte[] input) {
    * determined by the convention of the number of bytes to be sent over the network which is
    * determined in a constant. -1 because we dont have the type here Bytes are interpreted as
    * NetworkByteOrder, therefore Big Endian
+   * 
+   * This function.
    */
   public static byte[] intToBytes(int input) {
```

Comment:
/wtd Refactor this file so the functions are not static but members of an instanceable class